js/components/portal-dashboard/overlay-student-response.tsx

```diff
@@ -21,11 +21,14 @@ export class StudentResponse extends React.PureComponent<IProps> {
 
     return (
       <div className={css.studentResponse} data-cy="overlay-student-response-area">
-        <StudentNavigator students={students}
-                          isAnonymous={isAnonymous}
-                          currentStudentIndex={currentStudentIndex}
-                          setCurrentStudent={setCurrentStudent}
-                          currentStudentId={currentStudentId}
+        <StudentNavigator
+          students={students}
+          isAnonymous={isAnonymous}
+          currentStudentIndex={currentStudentIndex}
+          setCurrentStudent={setCurrentStudent}
+          currentStudentId={currentStudentId}
+          nameFirst={true}
+          filler={false}
```

Comment:
Don't need to add filler={false} since it's optional, but I would vote to remove it and not try to add that div as part of the StudentNavigator component.